quasar-utils/javac-plugin/src/test/resources/net/corda/plugins/javac/quasar/test/TestCase1.java

```diff
@@ -0,0 +1,13 @@
+package net.corda.plugins.javac.quasar.test;
+
+import java.util.function.Consumer;
+import co.paralleluniverse.fibers.Suspendable;
+import co.paralleluniverse.fibers.SuspendExecution;
+
+public class TestCase1 {
```

Comment:
Not a very informative set of class names. What is each case testing?

Reply:
there are comments inside each test case's file and I don't believe that a single word (like a Java type identifier) is good enough to describe any of those tests

Reply:
> there are comments inside each test case's file and I don't believe that a single word (like a Java type identifier) is good enough to describe any of those tests

Then _don't_ use just a single word?

Reply:
you mean something like `CheckPluginCanDetectSuspendableMethodsMarkedUsingCusomAnnotations` ?